packages/one-app-bundler/README.md

```diff
@@ -49,6 +49,14 @@ module. The root module should include in its configuration
 `providedExternals`, which is an array of external dependencies to be bundled
 with it and provided to other modules.
 
+Modules shouldn't configure both `providedExternals` and `requiredExternals`.
+
+Any module with `requiredExternals` configured will be validated at runtime to ensure that
+the root module is in fact providing those `requiredExternals`, and will fail to load if it is
+not. For example, if your child module requires `^2.1.0` of a dependency but your root module provides `2.0.0`, One App will fail to load your child module as it does not satisfy the required semantic range.
```

Comment:
```suggestion
All modules `requiredExternals` are validated at runtime against the root modules list of `providedExternals`.  If the external dependency is not provided One App will throw an error. This will either result in the One App server not starting or, if it is already running, One App will not load that module. For example, if your child module requires `^2.1.0` of a dependency but your root module provides `2.0.0`, this will result in One App not loading that child module as the provided dependencies version does not satisfy the required semantic range.
This ensures that all of the listed dependencies features potentially required by the child module to work will be provided which could result in hard to debug bugs.
```